lib/board.rb

```diff
@@ -10,9 +10,40 @@ def initialize
          @cells[index] = Cell.new(index)
        end
      end
+
+		 ###########
+		 @all_cells = []
+		 @all_cells = @cells.keys
```

Comment:
This is redundant, I think we just need the second line, since it's making an array :)